src/server/scripts/Commands/cs_gm.cpp

```diff
@@ -210,7 +210,7 @@ class gm_commandscript : public CommandScript
         }
         else
         {
-            _player->AddAura(VISUAL_AURA, _player);
```

Comment:
But in places like this, when we add the aura to the same "caster" I think this change is good